code/modules/cooking/machinery/cooking_machines/_appliance.dm

```diff
@@ -122,16 +117,10 @@
 		to_chat(user, "You can't reach [src] from here.")
 		return
 
-	if (stat & POWEROFF)//Its turned off
-		stat &= ~POWEROFF
-		use_power = 1
-		user.visible_message("[user] turns [src] on.", "You turn on [src].")
-
-	else //Its on, turn it off
-		stat |= POWEROFF
-		use_power = 0
-		user.visible_message("[user] turns [src] off.", "You turn off [src].")
-
+	stat ^= POWEROFF // Toggles power
+	use_power = !(stat & POWEROFF) && 2 // If on, use active power, else use no power
+	if(user)
+		user.visible_message("[user] turns [src] [use_power ? "on" : "off"].", "You turn [use_power ? "on" : "off"] [src].")
```

Comment:
```suggestion
		user.visible_message("[user] turns [src] [use_power ? "on" : "off"].", "You turn [src] [use_power ? "on" : "off"].")
```

Reply:
"you turn on the stove" and "you turn the stove on" are equivalent, tbh, but alright